src/ExpressionEvaluator/Core/Source/FunctionResolver/VisualBasic/Scanner.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.Utilities;
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator
+{
+    internal sealed partial class MemberSignatureParser
```

Comment:
The C# and VB scanners are similar. I decided not to share the code though since the scanners are quite simple and since the implementations might diverge in the future (if we decide to handle full-width characters in VB for instance). We can certainly merge the two later if reasonable.